src/compiler/backend/riscv32/code-generator-riscv32.cc

```diff
@@ -1923,13 +1920,6 @@ CodeGenerator::CodeGenResult CodeGenerator::AssembleArchInstruction(
       __ vmv_vx(dst, zero_reg);
       break;
     }
-    case kRiscvS128Load32Zero: {
```

Comment:
Should not delete because the result of lw would be used in the following instruction to set the SIMD128 register.

Reply:
added in latest commit.